index.js

```diff
@@ -56,43 +61,43 @@ module.exports = (function (){
 
   return {
     api: async () => {
-      return await apiContract.at(Chain.API());
+      return await apiContract.at('0x7c728214be9a0049e6a86f2137ec61030d0aa964');
```

Comment:
These are _not_ hardcoded. They might change, which is the reason why we are calling them externally. These should be called from `@mybit/contracts`, not `@mybit/chain`, that's part of the issue, but we do not hardcode them.